critiquebrainz/db/review.py

```diff
@@ -40,6 +40,20 @@ def to_dict(review, confidential=False):
     return review
 
 
+def get_top_languages():
```

Comment:
I would really like if you could write a `docstring` for this function. For writing `docstring`, see https://github.com/metabrainz/guidelines/blob/master/Python.md#docstrings. Also, consult how docstrings are writtten for other functions in `db/review.py`.

Reply:
Good to write a docstring in the following way:
```
"""
Returns:
    List of the top ten languages (only the language codes) in CritiqueBrainz.

"""
```